src/vs/base/browser/ui/positronComponents/splitters/verticalSplitter.tsx

```diff
@@ -396,24 +382,22 @@ export const VerticalSplitter = ({
 				}}
 				onPointerEnter={sashPointerEnterHandler}
 				onPointerLeave={sashPointerLeaveHandler}
-				onPointerDown={sashPointerDownHandler}
-				onPointerMove={sashPointerEventHandler}
-				onLostPointerCapture={sashPointerEventHandler}
+				onPointerDown={pointerDownHandler}
```

Comment:
The `pointermove` and `lostpointercapture` events are no longer being managed via React. The event listeners for these are added directly to the element in the DOM.